lib/std/Build/Step/Compile.zig

```diff
@@ -915,7 +927,7 @@ fn make(step: *Step, prog_node: *std.Progress.Node) !void {
     const cmd = switch (self.kind) {
         .lib => "build-lib",
         .exe => "build-exe",
-        .obj => "build-obj",
+        .obj, .pch => "build-obj",
```

Comment:
if it's building a precompiled header, it's not exactly building an object is it?